docs/alerts/scheduled-searches/index.md

```diff
@@ -1,13 +1,12 @@
 ---
 slug: /alerts/scheduled-searches
 title: Scheduled Searches
-sidebar_label: Scheduled Searches
 description: You can schedule log searches to send alerts.
 ---
 
```

Comment:
You need to update the titles of the cards in this article to match the titles of the articles, and you need to put a few in the correct order.

Reply:
good catch! will do this in my next PR push.